modules/dnn/CMakeLists.txt

```diff
@@ -6,16 +6,36 @@ if(DEFINED BUILD_opencv_dnn AND NOT BUILD_opencv_dnn)
   return()
 endif()
 
-include(${OpenCV_SOURCE_DIR}/cmake/OpenCVFindLibProtobuf.cmake)
-if(NOT Protobuf_FOUND)
-  ocv_module_disable(opencv_dnn)
+ocv_option(${the_module}_USE_PROTOBUF "Force use of Protobuf library instead OpenCV's protobuf parser" OFF)
+
+# One and only way if dnn module will be disabled - failed to link to system protobuf.
+if (DEFINED BUILD_PROTOBUF AND NOT BUILD_PROTOBUF)
```

Comment:
`if` should be used without space in CMake here and below

Reply:
Replaced